metadata_backend/api/operators.py

```diff
@@ -39,20 +39,22 @@ def __init__(self, db_name: str, content_type: str, db_client: AsyncIOMotorClien
         self.db_service = DBService(db_name, db_client)
         self.content_type = content_type
 
-    async def create_metadata_object(self, schema_type: str, data: Union[Dict, str]) -> Dict:
+    async def create_metadata_object(self, schema_type: str, data: Union[Dict, str]) -> Any:
         """Create new metadata object to database.
 
         Data formatting and addition step for JSON or XML must be implemented
         by corresponding subclass.
 
         :param schema_type: Schema type of the object to create.
         :param data: Data to be saved to database.
-        :returns: Tuple of Accession id for the object inserted to database and its title
+        :returns: Dict of Accession id for the object inserted to database and its title
```

Comment:
Nice if this matches the return type.

Reply:
Actually noticed many of these were wrong and tried to fix them